README.md

```diff
@@ -8,40 +7,44 @@ distinguishes itself with the following features:
 - Totally transparent field encryption: the value will only be encrypted in the database, never in the value
 - Unit testing
 
+## Integrations
+
+The package supports the following integrations:
+
+- Laravel
+
+## Upgrading
+
+If you're upgrading from a previous version you can find some help with that in [the upgrading guide](UPGRADING.md).
+
 ## Installation
-Add `51systems/doctrine-encrypt` to your Composer manifest.
 
-```js
-{
-    "require": {
-        "51systems/doctrine-encrypt": "~5.0"
-    }
-}
+```bash
+composer require 51systems/doctrine-encrypt
 ```
 
 ## Configuration
-### Using ZF2
-Check out the doctrine-encrypt-module at https://github.com/51systems/doctrine-encrypt-module
 
-### Manually
-Add the event subscriber to your entity manager's event manager. Assuming `$em` is your configured entity manager:
+### Laravel
```

Comment:
You seem to have taken a library which was general purpose and turned in into a Laravel specific module. I'd suggest adding instructions to use this library manually as well as within Laravel to ensure that users who currently use the Zend module or have pulled this into another framework can adapt their code.